public/datasets/exoplanets.json

```diff
@@ -1,71 +1,71 @@
 {
-    "card": {
-      "description": "This dataset is a list of exoplanets, which are planets orbiting other stars. With this data, it's possible to estimate the size of an exoplanet as well as other characteristics.",
-      "source": "https://exoplanetarchive.ipac.caltech.edu/",
-      "lastUpdated": "2021",
-      "context": {
-        "createdBy": "California Institute of Technology",
-        "potentialUses": "Estimate the size or mass of an exoplanet based on the size, age or other aspects of the star it orbits.",
-        "potentialMisuses": "Trying to make guesses as to whether a planet could support life."
-      }
+  "name": "exoplanets",
+  "card": {
```

Comment:
This looks like a big diff, but I just fixed misaligned indentation. Same for jeans and songs.

Reply:
Looks good with https://github.com/code-dot-org/ml-playground/pull/194/files?diff=unified&w=1